docs/run/primitives-examples.mdx

```diff
@@ -39,13 +39,21 @@ import numpy as np
 from qiskit.circuit.library import IQP
 from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
 from qiskit.quantum_info import SparsePauliOp, random_hermitian
-from qiskit_ibm_runtime import EstimatorV2 as Estimator, QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+from qiskit_ibm_runtime.fake_provider import FakeSherbrooke
 
-service = QiskitRuntimeService()
-backend = service.least_busy(operational=True, simulator=False, min_num_qubits=127)
+# Run the job locally using FakeSherbrooke
+backend = FakeSherbrooke()
+
+# Uncomment the following lines to run on a real system:
+# from qiskit_ibm_runtime import QiskitRuntimeService
+# service = QiskitRuntimeService()
+# backend = service.least_busy(operational=True, simulator=False, min_num_qubits=127)
 estimator = Estimator(backend)
 
-n_qubits = 127
+# Use fewer qubits when testing locally
+# n_qubits = 127
+n_qubits = 4
```

Comment:
Rather than making this change, what about truncating the output? For example,

```
# Output truncated for brevity
{'0111111": 41, '001111': 2, ...}
```

Reply:
> Rather than making this change, what about truncating the output? For example,
> 
> ```
> # Output truncated for brevity
> {'0111111": 41, '001111': 2, ...}
> ```

You mean why change from 12 to 4 qubits, as @ElePT suggested?  That's because she pointed out that there's not any real difference in 12 vs 4 qubits, and we can show the full output.  Or are you asking why change from 127 to 4 qubits?  I have it running on fewer qubits because I'm running it locally.  It won't run locally with 127 qubits.